bundle/compliance/cis_gcp/rules/cis_4_3/test.rego

```diff
@@ -0,0 +1,33 @@
+package compliance.cis_gcp.rules.cis_4_3
+
+import data.cis_gcp.test_data
+import data.compliance.policy.gcp.data_adapter
+import data.lib.test
+
+test_violation {
+	eval_fail with input as rule_input({})
```

Comment:
`block-project-ssh-keys` default value is `false`?

Reply:
by default this key,value will not exist.

Reply:
"By Default Block Project-wide SSH keys is not enabled."